INSTALL.md

```diff
@@ -0,0 +1,126 @@
+
+The above allows you to develop at NERSC and run pytest.  Your workflow as a developer would be to make your changes, do a `python setup.py install` then `pytest` to make sure your changes did not break any tests.
+
+If you are a DESC member you may also add to your CLMM environment the GCR and GCRCatalog packages to access the DC2 datasets at NERSC. To run the DC2 example notebooks provided in CLMM, the following need to be installed in your CLMM environment at NERSC. Once in your CLMM environment (`source activate clmmenv`), run
+
+```bash
+    pip install pandas
+    pip install pyarrow
+    pip install healpy
+    pip install h5py
+    pip install GCR
+    pip install https://github.com/LSSTDESC/gcr-catalogs/archive/master.zip
+    pip install https://github.com/yymao/FoFCatalogMatching/archive/master.zip
+```
+
+To open up a notebook from NERSC in your browser, you will need to go to the [nersc jupyter portal](https://jupyter.nersc.gov) and sign in. You will need to make this conda environment available in the kernel list:
+
+```bash
+    python -m ipykernel install --user --name=conda-clmmenv
+```
+
+Clicking on the upper right corner of the notebook will provide options for your kernel.  Choose the kernel `conda-clmmenv` that you just created.  You will need to do a temporary install of both cluster_toolkit and clmm in the first cell of your jupyter notebook:
+
+```python
+
+def install_clmm_pipeline(upgrade=False):
+    import sys
+    try:
+        import clmm
+        import cluster_toolkit
+        installed = True
+    except ImportError:
+        installed = False
+    if not upgrade:
+        print('clmm is already installed and upgrade is False')
+    else:
+        !{sys.executable} -m pip install --user --upgrade git+https://github.com/tmcclintock/cluster_toolkit.git
+        !{sys.executable} -m pip install --user --upgrade git+https://github.com/LSSTDESC/CLMM
+install_clmm_pipeline(upgrade=True)  # Comment this if you do not need to adjust your environment, but this is useful in cori
+
+```
+
+## Making a local copy of CLMM <a name="making_a_local_copy_of_clmm"></a>
+
+As a newcomer, you likely will not have edit access to the main CLMM repository.
+Without edit privileges, you won't be able to create or push changes to branches in the base repository. You can get around this by creating a [fork](https://help.github.com/articles/fork-a-repo/), a linked copy of the CLMM repository under your Github username. You can then push code changes to your fork which can later be merged with the base repository.
+
+To create a fork, navigate to the [CLMM home page](https://github.com/LSSTDESC/CLMM) and click 'Fork' in the upper right hand corner. The fork has been created under your username on Github's remote server and can now be cloned to your local repository with
+
+```bash
+    git clone git@github.com:YOUR-USERNAME/CLMM.git
```

Comment:
Do I need to do `cd CLMM` before the next step?

Reply:
Yes, fixed.